test/common/filesystem/directory_test.cc

```diff
@@ -48,6 +48,15 @@ class DirectoryTest : public testing::Test {
     }
   }
 
+  void addFileWithContents(absl::string_view file_name, absl::string_view contents) {
+    const std::string full_path = absl::StrCat(dir_path_, "/", file_name);
+    {
+      std::ofstream file(full_path);
+      file << contents;
```

Comment:
should we check for errors here, even with (say) an EXPECT_* macro?

Reply:
Added test-asserts, and one in the pre-existing similar spot too.